app/controllers/api/v1/private/publishers_controller.rb

```diff
@@ -0,0 +1,78 @@
+class Api::V1::Private::PublishersController < Api::V1::Private::BaseController
```

Comment:
I'd argue that you get rid of the `private` path, as it is already inferred.

Reply:
Good point, done in 0f23bb9